03-algorithms-sorting/bubble_sort.rb

```diff
@@ -0,0 +1,21 @@
+class BubbleSort
```

Comment:
Thanks. That is mostly based on the Bloc's code from the checkpoint. This is what I coded from scratch. Would you please review this code too?

```ruby
def bubble_sort(array)
  swap = 0
  l = array.length
  for i in 0..l - 2
    if array[i] > array[i+1]
      swap += 1
      temp = array[i]
      array[i] = array[i+1]
      array[i+1] = temp
    end
  end

  if swap > 0
    bubble_sort(array)
  end
  array
end
```